internal/local/structure_v2.go

```diff
@@ -534,6 +562,70 @@ func writeSync(rootDir string, sync SyncStructure) error {
 	)
 }
 
+func writeSchemas(rootDir string, schemas []map[string]interface{}) error {
+	dir := filepath.Join(rootDir, NameDataSources)
+	if err := os.MkdirAll(dir, os.ModePerm); err != nil {
+		return err
+	}
+	for _, schema := range schemas {
+		schemaBody := schema[NameSchema]
+		if schemaBody == nil {
+			schemaBody = map[string]interface{}{}
+		}
+		dataSchema, err := MarshalJSON(schemaBody)
+		if err != nil {
+			return err
+		}
+		relationships := schema[NameRelationships]
+		if relationships == nil {
+			relationships = map[string]interface{}{}
+		}
+		dataRelationships, err := MarshalJSON(relationships)
+		if err != nil {
+			return err
+		}
+
+		metadata := schema["metadata"].(map[string]interface{})
```

Comment:
if you're going to use `.(map[string]interface{})` then you should accompany it with the `, ok` syntax to avoid potential panics